src/components/NavBar/NavigationBar.tsx

```diff
@@ -43,45 +59,52 @@ function NavigationBar(): JSX.Element {
     };
   }, []);
 
-  // Toggles the navigation
-  function toggleNav(): void {
-    setIsNavVisible(!isNavVisible);
-  }
-
-  /** Toggle the language of the website between English and Spanish. Close menu bar after toggling if on mobile. */
-  function toggleLanguage(): void {
-    i18n.changeLanguage(i18n.language === 'en' ? 'es' : 'en');
-    if (isMobile) {
-      toggleNav();
+  /**
+   * Sets isScrolled depending on how far the user has scrolled
+   */
+  function handleScroll(): void {
+    const scrollThreshold = 20;
+    const offset = window.scrollY;
+    if (offset > scrollThreshold) {
+      setIsScrolled(true);
+    } else {
+      setIsScrolled(false);
     }
   }
 
-  const {t, i18n} = useTranslation('menu');
+  /**
+   * Event listener for scrolling
+   */
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll);
+    return function (): void {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  });
+
   return (
-    <div>
-      <header className="Navigation_Header">
-        {/* Logo in Spanish and English are the same */}
-        <img src={cehatLogo} className="Logo" alt="Logo" />
-        {isNavVisible && (
-          <nav>
-            <a href="/">{t('home')}</a>
-            <a href="/health">{t('healthInfo')}</a>
-            <a href="/about">{t('about')}</a>
-            <button id="changeLanguage" onClick={toggleLanguage}>
-              <Icon as={FaGlobeAmericas} id="globe" title={t('globeIcon')} />
-              {t('changeLanguage')}
-            </button>
-          </nav>
-        )}
-        <button onClick={toggleNav} className="Burger">
-          <Icon
-            as={FaBars}
-            className="menu-icon"
-            aria-label={t('menuButton')}
-          />
-        </button>
-      </header>
-    </div>
+    <Flex
+      align="center"
+      justify="space-between"
+      wrap="wrap"
+      w="100%"
+      mb={2}
+      p={3}
+      background="teal"
```

Comment:
The new color scheme feels a bit dark and depressing to me. The blue felt more welcoming and inviting. I think we should definitely run any change to the major UI (color scheme, logo, etc) by the liasions first.